src/util.ts

```diff
@@ -129,3 +131,24 @@ export async function download(url: string): Promise<string> {
     req.end();
   });
 }
+
+async function getFiles(filePath :string) {
```

Comment:
Please annotate this with the appropriate return type (I think it's `string[]`?)
```suggestion
async function getFiles(filePath: string): string[] {
```